app/models/concerns/image_file.rb

```diff
@@ -61,6 +91,7 @@ def reset_image_attributes
     self.height = nil
     self.width = nil
     self.size = nil
+    self.status = :created
```

Comment:
This seemed like an oversight - but we probably didn't notice, since you normally run `copy_media` right away after changing the original_url.